src/components/UserProfile/application/PendingApplications.tsx

```diff
@@ -0,0 +1,239 @@
+import DeleteIcon from '@material-ui/icons/Delete';
+import InfoIcon from '@material-ui/icons/Info';
+import React, { FC, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link as RouterLink } from 'react-router-dom';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Link from '@material-ui/core/Link';
+import { Typography } from '@material-ui/core';
+import {
+  refetchUserApplicationsQuery,
+  useChallengeNameIdQuery,
+  useDeleteUserApplicationMutation,
+  useEcoverseNameIdQuery,
+  useOpportunityNameIdQuery,
+  useUserProfileApplicationsQuery,
+} from '../../../hooks/generated/graphql';
+import { APPLICATION_STATE_NEW, APPLICATION_STATE_REJECTED } from '../../../models/constants';
+import { ApplicationResultEntry, User } from '../../../models/graphql-schema';
+import { createStyles, useApolloErrorHandler, useNotification } from '../../../hooks';
+import Tag from '../../core/Tag';
+import IconButton from '../../core/IconButton';
+import Card from '../../core/Card';
+import getApplicationWithType, {
+  ApplicationType,
+  ApplicationWithType,
+} from '../../../utils/application/getApplicationWithType';
+import { buildChallengeUrl, buildEcoverseUrl, buildOpportunityUrl } from '../../../utils/urlBuilders';
+import ApplicationDialogWrapper from './ApplicationDialogWrapper';
+import Tooltip from '@material-ui/core/Tooltip';
+
+const useStyles = createStyles(theme => ({
+  listDetail: {
+    padding: theme.spacing(1),
+    marginTop: theme.spacing(1),
+    backgroundColor: theme.palette.neutralLight.main,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+  labels: {
+    display: 'flex',
+    justifyContent: 'end',
+    gap: theme.spacing(0.5),
+  },
+  row: {
+    padding: `${theme.spacing(0.5)}px 0`,
+  },
+  noPadding: {
+    padding: 0,
+  },
+  statusRow: {
+    display: 'flex',
+    gap: theme.spacing(0.5),
+  },
+  capitalize: {
+    textTransform: 'capitalize',
+  },
+}));
+
+interface Props {
+  user?: User;
+  canEdit?: boolean;
+}
+
+const PendingApplications: FC<Props> = ({ user, canEdit = true }) => {
```

Comment:
This can be extracted into a container with business logic. Loading data, actions etc. example: `OpportunityMembers`.